toolchain/gcc/initial/Makefile

```diff
@@ -19,12 +19,10 @@ endef
 
 define Host/Install
 	+$(GCC_MAKE) $(HOST_JOBS) -C $(GCC_BUILD_DIR) \
-		prefix="$(TOOLCHAIN_DIR)/initial" \
```

Comment:
should we keep the `prefix=` and just remove "initial" from it??

(and in binutils too)

Reply:
Definitely not ;) It's passed to ./configure, there's no need to override it for the `make` call as the value is compiled in.